### PR TITLE
[WIP] New OSE job 

### PR DESCRIPTION
**Pull request by jhadvig**
@stevekuznetsov ptal

### CODE REVIEW

File: sjb/config/test_cases/test_pull_request_ose_new.yml [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  default_value: "<none>"
+- name: "MERGE_SEVERITY"
+  description: ""
+  default_value: ""
+- name: "MERGE"
+  description: ""
+  default_value: ""
+provision:
+  os: "rhel"
+  stage: "base"
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_pull_request_ose_new.yml]: we need stage `bare` on this one

File: sjb/config/test_cases/test_pull_request_ose_new.yml [hunk 2]
```diff
@@ -0,0 +1,161 @@
+---
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_pull_request_ose_new.yml]: I was expecting just to see a shell script -- we can't generate and push out to that jenkins master

File: sjb/config/test_cases/test_pull_request_ose_new.yml [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+provision:
+  os: "rhel"
+  stage: "base"
+  provider: "aws"
+actions:
+  - type: "host_script"
+    title: "clone ose"
+    timeout: 300
+    script: |-
+      if $MERGE; then
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_pull_request_ose_new.yml]: do all this on the remote after you have cloned and pushed

File: sjb/config/test_cases/test_pull_request_ose_new.yml [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        approve.sh 'ose' "${BRANCH}" "${MERGE_SEVERITY}"
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_pull_request_ose_new.yml]: `sjb` will put this in automatically

**Comment by kargakis** [comment]
What's the status of this?

**Comment by jhadvig** [comment]
Closing this cause the changis have already been made to the https://github.com/openshift/ci-config/blob/master/jobs/test_pull_requests_ose.sh file

**Pull request closed without merging**